README

```diff
@@ -1327,23 +1330,69 @@ of the nullmodem connection. These are all parameters:
 Example: Be a server listening on TCP port 5000.
    serial1=nullmodem server:<IP or name of the server> port:5000 rxdelay:1000
 
-BBS Gaming
-----------
-DOSBox's serial interface can emulate a telephone modem, which allows original
-DOS terminal applications to either dial or host a BBS on the Internet 
-via the Telnet protocol.
 
-First, configure DOSBox with a serial port emulating a modem:
+BBS Support: softmodem connection over TCP
+------------------------------------------
+DOSBox emulates a hardware modem allowing one to either host or dial a BBS
+across the Internet.
+
+To serve a BBS behind a Telnet interface:
+
+ 1) Configure DOSBox's serial port as follows:
+
+    [serial]
+    serial1 = modem telnet:1 listenport:2323
+
+    A port greater than 1024 allows you to run DOSBox as a normal user instead
+    of requiring root or administrator rights.
+
+ 2) Configure your machine to use a static IP address or be assigned a static
+    IP via DHCP, which can typically be configured in your router.
+
```

Comment:
Thanks, yes, I will improve this!

Reply:
This wasn't clear enough; the intent is that the machine inside the network (behind the router/firewall) should have a static IP.  A lot of people runinng Windows just use DHCP and that makes a port-foward unreliable because typically the rule is IP-based (and not hostname-based); so it will work on day one, but 24-hours later their PC might get a new IP and then their BBS goes "offline". 

We don't want to go overboard (and out of scope) in the README, so I'll try to balance this out.